### PR TITLE
Undocumented Missing Mirror Parts of https://github.com/tgstation/tgstation/pull/59007

### PR DESCRIPTION
**Pull request by nevimer**
… udders.dm) (#59007)

* Update prefixes.dm

* widening the scope to be fixing master in general # Conflicts:
#	code/__DEFINES/dcs/signals.dm
#	code/datums/components/udder.dm

### CODE REVIEW

**Comment by nevimer** [comment]
also hard del

**Comment by nevimer** [comment]
if you want it go ahead and reopen, im not fighting stalebot.

**Comment by nevimer** [comment]
FEATURE FREEZE FEBRUARY, BEST TIME TO OPEN MIRRORS AND SORT THINGS OUT

File: code/datums/components/udder.dm [hunk 1]
```diff
@@ -137,6 +137,10 @@
 		START_PROCESSING(SSobj, src)
 	RegisterSignal(udder_mob, COMSIG_HOSTILE_PRE_ATTACKINGTARGET, PROC_REF(on_mob_attacking))
 
+/obj/item/udder/gutlunch/Destroy()
+	. = ..()
+	UnregisterSignal(udder_mob, COMSIG_HOSTILE_PRE_ATTACKINGTARGET)
+
```
Comment (ghost) [inline code/datums/components/udder.dm]: TG no longer has this on their latest master, this entire chunk needs removed

Comment (tf-4) [inline code/datums/components/udder.dm]:
```suggestion
```

**Review by tf-4 — APPROVED** [review]

**Pull request merged**